.openpublishing.redirection.json

```diff
@@ -3,6 +3,26 @@
         {
             "source_path_from_root": "/docs/storage/quickstart-azure-storage.md",
             "redirect_url": "/dotnet/aspire/storage/azure-storage"
+        },
+        {
+            "source_path_from_root": "/docs/storage/azure-storage.md",
+            "redirect_url": "/dotnet/aspire/storage/azure-storage-components"
+        },
+        {
+            "source_path_from_root": "/docs/database/quickstart-sql-server.md",
+            "redirect_url": "/dotnet/aspire/database/sql-server-components"
+        },
+        {
+            "source_path_from_root": "/docs/get-started/quickstart-build-your-first-aspire-app.md",
+            "redirect_url": "/dotnet/aspire/get-started/build-your-first-aspire-app"
+        },
+        {
+            "source_path_from_root": "/docs/messaging/quickstart-messaging.md",
+            "redirect_url": "/dotnet/aspire/messaging/messaging-components"
+        },
+        {
+            "source_path_from_root": "/docs/caching/quickstart-caching.md",
+            "redirect_url": "/dotnet/aspire/caching/caching-components"
         }
```

Comment:
My only thought here is whether these new URLs might be too general to be future proof, for example if we want to add similar tutorials for messaging or caching as we support additional services/providers. I suppose we could name the new ones more specifically but just something to consider.

Reply:
I think the file name is fine, they'd just have to be more nested in the TOC maybe?

Database/
  SQL/
    Raw
    EF Core
  PostgreSQL
    Raw
    EF Core